src/shared/components/Tooltip/Tooltip.tsx

```diff
@@ -17,7 +17,7 @@ const applyMaxSize = {
       ...state.styles.popper,
       display: 'flex',
       placeItems: state.placement.includes('bottom') ? 'flex-start' : 'flex-end',
-      height: `${height - 24}px`,
+      height: state.placement.includes('bottom') ? `${height - 24}px` : undefined,
```

Comment:
I understand this change, but I think it also introduces a couple drawbacks. 

This height value obviously doesn't work great for tooltips arranged to the left or right of the reference element, but opting out entirely means we allow tooltip content to potentially extend off screen. The `height` value itself it also still necessary for tooltips with "top" placements. 

Applying some of these styles is admittedly a bit cumbersome, because we can't directly apply the `maxHeight` to the actual styled tooltip (these styles are applied via its' own modifier function to the wrapper element around the tooltip, and are used to position the tooltip within it).

I gave tweaking the styles a shot myself to see if we could improve things somewhat, and this is what I came up with:

```jsx
    const isVerticallyArranged =
      state.placement.includes('bottom') || state.placement.includes('top');

    state.styles.popper = {
      ...state.styles.popper,
      display: 'flex',
      ...(isVerticallyArranged && {
        placeItems: state.placement.includes('bottom') ? 'flex-start' : 'flex-end',
      }),
      height: isVerticallyArranged ? `${height - 24}px` : undefined,
      maxHeight: `${height - 24}px`,
    };
  },
```

Here's the before/after of the change:

| Before | After |
|-|-|
| <img width="485" alt="before" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/731daabe-af2f-429f-9191-decb78788a3e"> | <img width="454" alt="Screenshot 2024-06-25 at 12 16 32 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/5466362e-c8dc-48cb-aa4c-4cf0750dc0e8"> |

... and tooltip content that _doesn't_ fill the max height is still positioned correctly.

Reply:
I took a different approach to achieve the purpose behind the new styles that works for all placement options.

| Bottom | Top | Right |
|-|-|-|
| ![CleanShot 2024-06-26 at 09 31 16](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/5126dc87-28dd-4123-b7a2-495ee3c78e0b) | ![CleanShot 2024-06-26 at 09 34 02](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/722bca55-872b-4d42-b5ef-0ca74243a17b) | ![CleanShot 2024-06-26 at 09 31 06](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/71e59409-9e2f-44cc-bdfb-248906991138) |